src/tasks/minions/PrayerActivity/offeringActivity.ts

```diff
@@ -0,0 +1,83 @@
+import { Task, KlasaMessage } from 'klasa';
+
+import { saidYes, noOp, rand, roll } from '../../../lib/util';
+import { Time } from '../../../lib/constants';
+import { OfferingActivityTaskOptions } from '../../../lib/types/minions';
+import getUsersPerkTier from '../../../lib/util/getUsersPerkTier';
+import Prayer from '../../../lib/skilling/skills/prayer';
+import { channelIsSendable } from '../../../lib/util/channelIsSendable';
+import { SkillsEnum } from '../../../lib/skilling/types';
+
+export default class extends Task {
+	async run({ boneID, quantity, userID, channelID }: OfferingActivityTaskOptions) {
+		const user = await this.client.users.fetch(userID);
+		const currentLevel = user.skillLevel(SkillsEnum.Prayer);
+
+		const Bury = Prayer.Bones.find(Bury => Bury.inputId === boneID);
```

Comment:
why is this called `Bury`? it should be `bones`, the casing might be uppercased in others but we should start doing it properly camelcased.